lib/contracts/flowdock-thread.ts

```diff
@@ -11,7 +11,7 @@ export const flowdockThread: ContractDefinition = {
 			type: 'object',
 			properties: {
 				data: {
-					required: ['title', 'id', 'flow'],
+					required: ['title', 'id', 'flow', 'urls.items'],
```

Comment:
```suggestion
					required: ['title', 'id', 'flow', 'url'],
```

Assuming I'm right and we only need to keep a single string `url` and not an array of strings. Even if we do want to allow for multiple URLs, you wouldn't use dot notation, it would just be `urls`.